packages/widgets/src/components/NotificationCard/types.ts

```diff
@@ -0,0 +1,12 @@
+import { IMapIcons } from '@ssa-ui-kit/core';
+
+export interface INotificationCardProps {
+  title: string | JSX.Element;
+  text: string | JSX.Element;
```

Comment:
Could we use React.ReactNode instead of "string | JSX.Element"?

Reply:
done